secrets-provider-chart/ci/test-values-template.yaml

```diff
@@ -0,0 +1,25 @@
+# K8s permission defaults
+rbac:
+  roleName: {{ SECRETS_PROVIDER_ROLE }}
+  roleBindingName: {{ SECRETS_PROVIDER_ROLE_BINDING }}
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: {{ SERVICE_ACCOUNT_CREATE }}
+    # The name of the service account to use
+    name: {{ SERVICE_ACCOUNT }}
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
```

Comment:
what's this tag? is this the tag of the image? do we want it static?

Reply:
this tag is something that for each release of a helm chart we would update and ship off to the customer. So I would imagine for Rotation Milestone the version here will be 2.0.0.

Now that I think about it, this sort of limits us from releasing a new secrets provider major.minor.patch version without delivering a new Helm to the customer.. What do you think?

Reply:
I suggest to put both image and tag field contents to values.yaml - see example in helm chart I have provided. This way of the user wants he can control them. The defaults of these field in values.yaml should be as you said Sigal

Reply:
It already exists there 
```
secretsProvider:
  image: cyberark/secrets-provider-for-k8s
  tag: 1.0.0
  name: cyberark-secrets-provider
```
The ci folder is for our pipeline to run our tests